rfd/0159-password-status.md

```diff
@@ -0,0 +1,128 @@
+---
+authors: Bartosz Leper (bartosz.leper@goteleport.com)
+state: draft
+---
+
+# RFD 0159 - Password Status
+
+## Required Approvers
+
+- Engineering:
+- Security:
+- Product:
+
+## What
+
+We want to improve the UX of our flows related to changing passwords and management of authentication devices (both MFA and passwordless). The following new features must be supported:
+
+- The user must be able to set password having only passwordless credentials configured.
+- We should do our best to visualize the status of user's password. Unfortunately, due to past design decisions, we will not be able to do it in every case; see the [Details](#details) section.
```

Comment:
Consider this alternate wording:

- We should do our best to indicate which authentication methods a user has configured.